content/pages/beta-enrollment/claim-increase.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Try Our New Beta Tools
+layout: page-react.html
+custom_react_root: true
```

Comment:
Do we have any documentation this should be added to?

Reply:
Not sure, I'll go take a look.